MAUI/Segmented-Control/customization.md

```diff
@@ -0,0 +1,395 @@
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Customization of .NET MAUI Segmented (SfSegmented) control
+The .NET MAUI Segmented control allows you to customize the background, text color, selection style, and much more.
+
+## Customizing the border color
+You can use `Stroke` property of `SfSegmentedControl`, to customize the border color of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                Stroke="Red">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.Stroke = Colors.Red;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the border thickness
+
+You can use `StrokeThickness` property of `SfSegmentedControl`, to customize the border thickness of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                StrokeThickness="1">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.StrokeThickness = 1;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the corner radius
+
+You can use `CornerRadius` property of `SfSegmentedControl`, to customize the corner radius of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                CornerRadius="5">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.CornerRadius = 5;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment width
```

Comment:
Add separate md file for Layout

move
Segmented width, Height, Visible segemnts 

Add Note for Segment width and Visible count

Reply:
Added separate md file for Layout and done changes